src/App.js

```diff
@@ -1,25 +1,47 @@
 import React from 'react';
-
 import './App.scss';
+import Clock from './components/Clock';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  render() {
+    const { isClockVisible } = this.state;
+    const newName = Math.floor(Math.random() * 100);
```

Comment:
better to create some method (setRandomName or another name) and create const newName inside that method